src/index.html

```diff
@@ -7,6 +7,343 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="nav">
+        <div class="nav__logo">
+          <img src="./images/logo.png" alt="Moyo header logo">
+        </div>
+
+        <nav class="nav__list">
+          <a href="apple.html" class="nav__item is-active">
+            Apple
+          </a>
+          <a href="samsung.html" class="nav__item">
+            Samsung
+          </a>
+          <a href="smartphones.html" class="nav__item">
+            Смартфоны и телефоны
+          </a>
+          <a
+            href="notebook.html"
+            class="nav__item"
+            data-qa="nav-hover"
+          >
+            Ноутбуки и компьютеры
+          </a>
+          <a href="gadgets.html" class="nav__item">
```

Comment:
```suggestion
          <a href="#" class="nav__item">
```
У тебя нет этих страниц, можно поставить заглушку

Reply:
fixed with '#'